CHANGELOG.md

```diff
@@ -30,6 +30,77 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v9.0.2](https://github.com/shopsys/shopsys/compare/v9.0.1...v9.0.2) - 2020-08-27
+
+### [shopsys/framework]
+
+#### Changed
+
+- [#1750] Product::setProductCategoryDomains() fix
+- [#1941] better extensibility of article form type
```

Comment:
This is only PR from changed list that is here right, others are fixes

Reply:
I disaggre.

These PR are changes to me.
```
- [#1972] fixed removing friendly urls
- [#1986] design - admin - title button wrap bar only if contains buttons
```